server/webSocketServer/messageCreator.js

```diff
@@ -0,0 +1,8 @@
+const createMessage = (data, sendingUser) => (JSON.stringify({ data, from: { id: sendingUser.id, name: sendingUser.name } }));
+
+const createServerMessage = (data) => (JSON.stringify({ data, from: { id: 'server' } }));
```

Comment:
why arent you first doing from id and then the data ? header and body..

Reply:
nice ill change